### ISSUE TITLE
"@coroutine" decorator is deprecated since Python 3.8

### ISSUE THREAD

**Issue by devdjdjdj** [Tinche/aiofiles#81]
I get this warning when I run my code 

/python3.8/site-packages/aiofiles/os.py:10: DeprecationWarning: "@coroutine" decorator is deprecated since Python 3.8, use "async def" instead
    def run(*args, loop=None, executor=None, **kwargs):

**Comment by ewen-lbh** [comment]
I tried to remove the `@asyncio.coroutine` line and add an `async` keyword in front of `def` and it worked without any problems (for my test case at least), see the following diff

```diff
  def wrap(func):
-     @asyncio.coroutine
      @wraps(func)
-     def run(*args, loop=None, executor=None, **kwargs):
+     async def run(*args, loop=None, executor=None, **kwargs):
```

**Comment by samuelcolvin** [comment]
this can be closed v0.6 is released.
